acq4/devices/PMT/PMT.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.DAQGeneric import DAQGeneric
+
+class PMT(DAQGeneric, OptomechDevice):
```

Comment:
Why `PMT` instead of `PhotoDetector`? I imagine exactly the same code could be used to cover photodiodes. I also have need for a device like this when calibrating lasers, so thermopile detectors might be included as well.